DesktopModules/Vanjaro/Core/Library/Install/Uninstall.Migrate.SqlDataProvider

```diff
@@ -0,0 +1,2 @@
+delete from {databaseOwner}[{objectQualifier}PersonaBarMenu] where Identifier='Vanjaro'
+GO
```

Comment:
Why do we have this same uninstall in two places? Once here and once in Migrate?